src/styles/index.scss

```diff
@@ -1,3 +1,134 @@
-body {
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
```

Comment:
Default value, so you can just remove it
```suggestion
```

Reply:
I am fixed